test/parallel/test-child-process-kill-numeric.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const cat = common.spawnCat();
+
+cat.stderr.on('data', function(chunk) {
+  common.fail();
+});
+
+cat.on('exit', common.mustCall(function(code, signal) {
+  assert.strictEqual(code, null);
+  assert.strictEqual(signal, 'SIGTERM');
+}));
+
+assert.strictEqual(cat.killed, false);
+// numeric value of SIGTERM
```

Comment:
I think this could be `constants.os.signals.SIGTERM`, so the comment would be unnecessary.